packages/browser-agent-core/src/common/util/feature-flags.js

```diff
@@ -10,6 +10,7 @@ export function activateFeatures (flags, agentIdentifier) {
   var sharedEE = ee.get(agentIdentifier)
   if (!(flags && typeof flags === 'object')) return
   mapOwn(flags, function (flag, val) {
+    if (!val) return sharedEE.emit('block-' + flag, [])
     if (!val || activatedFeatures[flag]) return
     sharedEE.emit('feat-' + flag, [])
```

Comment:
So as to not introduce redundancy, could you remove the `feat-` that's leftover for STN, and inverse the logic in STN to correspond to `block-stn` _not_ being sent instead?

Reply:
nvm; ok with having the two "systems"